scripts/build_in_docker.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This script is used in the docker container to create a Cura AppImage.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/.."
+
+# Make sure that cura-build-environment is present
+if [[ -z "${CURA_BUILD_ENV_PATH}" ]]; then
+    echo "CURA_BUILD_ENV_PATH is not defined. Cannot find the installed cura build environment."
+    exit 1
+fi
+
+# Make sure that a directory for saving the resulting AppImage exists
+CURA_APPIMAGES_OUTPUT_DIR="${CURA_APPIMAGES_OUTPUT_DIR:-/home/ultimaker/appimages}"
+if [[ ! -d "${CURA_APPIMAGES_OUTPUT_DIR}" ]]; then
+    mkdir -p "${CURA_APPIMAGES_OUTPUT_DIR}"
+fi
+
+# Set up Cura build configuration in environment variables
+export CURA_VERSION_MAJOR="${CURA_VERSION_MAJOR:-4}"
+export CURA_VERSION_MINOR="${CURA_VERSION_MINOR:-1}"
+export CURA_VERSION_PATCH="${CURA_VERSION_PATCH:-99}"
+export CURA_VERSION_EXTRA="${CURA_VERSION_EXTRA:-docker}"
```

Comment:
So we'd have to update this with every release? I don't think that's a good idea. We're going to forget!

Normally we have this sort of thing set to version 0.0.0-master. This way, by default every developer who sets up their own build of Cura will not be linked to our Cura release versions, but that's fine. But we can still configure the version number from within Jenkins/Gitlab (or whichever CI system we'll be using by then).

Reply:
There's no need to update this script because you can configure those release variables such as version numbers via the environment variables. Although the default values should probably not be `4.1.99-docker`.

To create a new release, instead of directly running `scripts/build.sh`, you can run
```
# Suppose this release is 4.2.3, no extra version
CURA_VERSION_MAJOR=4 \
CURA_VERSION_MINOR=4 \
CURA_VERSION_PATCH=4 \
CURA_VERSION_EXTRA=4 \
...  \   # any other configurations
scripts/build.sh
```
The idea is that this script is reusable to create any release versions, and the only thing you need to do is to specify the parameters for a certain release.

Note that I took a shortcut in `scripts/build.sh` which does make those variables configurable. That should be changed.

Reply:
This is done in https://github.com/Ultimaker/cura-build/pull/210/commits/b629ae3fddb6db7ff3849a0eb491e40a50253427